src/core/ext/filters/client_channel/channel_connectivity.cc

```diff
@@ -33,22 +33,22 @@ grpc_connectivity_state grpc_channel_check_connectivity_state(
   /* forward through to the underlying client channel */
   grpc_channel_element* client_channel_elem =
       grpc_channel_stack_last_element(grpc_channel_get_channel_stack(channel));
-  grpc_exec_ctx exec_ctx = GRPC_EXEC_CTX_INIT;
+  grpc_core::ExecCtx _local_exec_ctx;
```

Comment:
_ prefixed names are against the style guide for some pretty good reasons... let's remove them globally before submission